test/backgroundWorker_test.cc

```diff
@@ -2,8 +2,8 @@
 #include <fcntl.h>
 #include <gflags/gflags.h>
 #include <rocksdb/file_system.h>
-#include <rocksdb/plugin/zenfs/fs/fs_zenfs.h>
-#include <rocksdb/plugin/zenfs/fs/zbd_zenfs.h>
+#include <third-party/zenfs/fs/fs_zenfs.h>
+#include <third-party/zenfs/fs/zbd_zenfs.h>
```

Comment:
IIRC, I put the `third-party/zenfs` in CMake's include_directories, maybe we should use `#include "fs/fs_zenfs.h"` here, please have a look. And for the coding style, we only use `<>` for system provided or third-party provided headers, for self-contained headers we use `“ ”`.